### PR TITLE
Adding requirements for Microverse challenges

### PR DESCRIPTION
**Pull request by tinatsina**
Added new files

.gitignore
style.css
index.html
linter.yml

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style/style.css">
+    <title>Microverse Hello</title>
+</head>
+<body>
+    <h1>
+        Hello Microverse!
+    </h1>
+</body>
+</html>
```
Comment (zieeco) [inline index.html:15]: **[REQUIRED CHANGES]**

- Kindly add the missing `.hintrc`, and ` linter file in your root folder from the [microverse linter](https://github.com/microverseinc/linters-config/tree/master/html-css) repository.

File: style/style.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: crimson;
+}
```
Comment (zieeco) [inline style/style.css:3]: **[REQUIRED CHANGES]**

- Kindly add the `.stylelintrc.json` linters files in your root folder from the [microverse linter](https://github.com/microverseinc/linters-config/tree/master/html-css) repository.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (zieeco) [inline .gitignore:1]: **[REQUIRE CHANGES]**

- [ ] Please, kindly add also the `node_modules/` directory in the `.gitignore` file.

File: README.md [hunk 4]
```diff
@@ -1 +1,47 @@
-# Microverse_Hello_World
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Microverse Hello World
+
+> First project of second day at Microverse.
```
Comment (zieeco) [inline README.md:5]: - [ ] **[REQUIRED CHANGES]**

- Please, add more description about the project to make your readme look more professional

File: README.md [hunk 5]
```diff
@@ -1 +1,47 @@
...
+
+## Built With
+
+- Markdown
```
Comment (zieeco) [inline README.md:9]: - [ ] **[REQUIRED CHANGES]**

- Let's update our `Buillt With` header to be more descriptive by doing  something like:

```
- HTML
- CSS
```

File: README.md [hunk 6]
```diff
@@ -1 +1,47 @@
...
+**Set up a linter for HTML and CSS**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+**Git**
+### Setup
+
+**Simply clone the GIT files**
```
Comment (zieeco) [inline README.md:22]: **[REQUIRE CHANGES]**

- We can improve the section by showing the user how to clone the repo by doing something like: 

```


To get a local copy up and running:

1. Clone this repository or download the Zip folder:

**``git clone https://github.com/tinatsina/Microverse_Hello_World.git``**

1. Navigate to the location of the folder in your machine:

**``you@your-Pc-name:~$ cd <folder>``**

```

File: README.md [hunk 7]
```diff
@@ -1 +1,47 @@
...
+
+## Authors
+
+👤 **Tinaye Tsinakwadi**
+
+- [GitHub](https://github.com/githubhandle)
+- [Twitter](https://twitter.com/twitterhandle)
+- [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
```
Comment (zieeco) [inline README.md:31]: **[REQUIRE CHANGES]**

- You can improve the section by filling it with your personal details and changing the `Authors` to `Author` as you worked alone on the project.

File: README.md [hunk 8]
```diff
@@ -1 +1,47 @@
...
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+
+## 📝 License
+
+No license
```
Comment (zieeco) [inline README.md:47]: **[REQUIRE CHANGES]**

- You can improve the section by adding the [microverse lincense](https://github.com/microverseinc/readme-template/blob/master/MIT.md) below:

This project is [MIT](https://github.com/microverseinc/readme-template/blob/master/MIT.md) licensed

File: README.md [hunk 9]
```diff
@@ -1 +1,47 @@
...
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
```
Comment (zieeco) [inline README.md:43]: **[REQUIRE CHANGES]**

- You can improve the section by adding the [microverse official website](https://bit.ly/MicroverseTN) below:

```
[Microverse](https://bit.ly/MicroverseTN)

```

File: README.md [hunk 10]
```diff
@@ -1 +1,55 @@
...
+
+## Getting Started
+
+**Set up a linter for HTML and CSS**
+
+To get a local copy up and running follow these simple example steps.
+
+1. Clone the repository or donwload a copy to your computer
+2. Go to the folder in your machine, this will be the root location
+3. For NPM use that folder as your root location
```
Comment (muneebulrehman) [inline README.md:21]: - [ ] optional
As suggested by previous reviewer, please use this type of template for project setup description


To get a local copy up and running:

1. Clone this repository or download the Zip folder:

**``git clone https://github.com/tinatsina/Microverse_Hello_World.git``**

1. Navigate to the location of the folder in your machine:

**``you@your-Pc-name:~$ cd <folder>``**

**Review by muneebulrehman — APPROVED** [review]
Hi @tinatsina , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳 
Congratulations! 🎉 

# Project Approved

## Highlights
- Followed previous reviewer's guidelines
- Improved the project from the last PR.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**